firewall/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.HomePage, name="home"),
+    path('addrule/',             views.AddRule,    name="addrule"),
+    path('updaterule/<str:pk>/', views.UpdateRule, name="updaterule"),
+    path('removerule/<str:pk>/',  views.RemoveRule,  name="removerule")
```

Comment:
Non-blocking: These two paths should be a single path with GET, PUT, and DELETE methods to follow RESTful API
For example: 
`    path('rule/<str:pk>/', views.rule_handler, name="rule_handler"),`

Reply:
remove rule will be implemented using javascript pop up instead of redirecting to another page for confirmation, soon